src/Boehm-GC.xml

```diff
@@ -14,8 +15,8 @@
             WARRANTY EXPRESSED OR IMPLIED. ANY USE IS AT YOUR OWN RISK.
          </p>
          <p>
-            Permission is hereby granted to use or copy this program for any
-            purpose, provided the above notices are retained on all copies.
+            Permission <optional>is hereby granted</optional> to use or copy this <alt name="program" match="program|software">program</alt> for any
+            purpose, <optional>is hereby granted without fee,</optional> provided the above notices are retained on all copies.

```

Comment:
I'm not sure this actually matches https://github.com/MariaDB/server/blob/11.6/libmysqld/lib_sql.cc with the present edits. Note the comma after "purpose" in Boehm-GC which isn't present in the MariaDB link.

Might be able to make that optional, but ugh... to @richardfontana's point, this does start to look a bit ugly...